client/reader/a8c/main.jsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+import { connect, useDispatch } from 'react-redux';
+import config from 'config';
+
+/**
+ * Internal dependencies
+ */
+import Stream from 'reader/stream';
+import { Button } from '@automattic/components';
+import SectionHeader from 'components/section-header';
+import { requestMarkAllAsSeen, requestMarkAllAsUnseen } from 'state/reader/seen-posts/actions';
+import { SECTION_A8C_FOLLOWING } from 'state/reader/seen-posts/constants';
+import { sectionHasUnseen } from 'state/reader/seen-posts/selectors';
+
+const A8CFollowing = ( props ) => {
+	const { translate } = props;
+	const dispatch = useDispatch();
+
+	const markAllAsSeen = () => {
+		dispatch( requestMarkAllAsSeen( { section: SECTION_A8C_FOLLOWING } ) );
+	};
+
+	const markAllAsUnSeen = () => {
+		dispatch( requestMarkAllAsUnseen( { section: SECTION_A8C_FOLLOWING } ) );
+	};
+
+	return (
+		<Stream { ...props } shouldCombineCards={ false }>
+			<SectionHeader label={ translate( 'Followed A8C Sites' ) }>
+				{ config.isEnabled( 'reader/seen-posts' ) && ! props.hasUnseen && (
+					<Button compact onClick={ markAllAsUnSeen }>
+						{ translate( 'Mark All as unseen' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Mark All as Unseen' )` *ES Score: 14*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/unseen-for-subsections)